pkg/api/testapi/testapi_test.go

```diff
@@ -46,9 +46,9 @@ func TestResourcePathWithPrefixForV1Beta3(t *testing.T) {
 	}
 }
 
-func TestResourcePathWithPrefixForV1Beta1(t *testing.T) {
-	if Version() != "v1beta1" {
-		// Skip the test if we are not testing v1beta1.
+func TestResourcePathWithPrefixForV1(t *testing.T) {
```

Comment:
we can combine these functions for V1Beta3 and V1, since apiVersion is the only difference in the expected string.
We required different functions for v1beta1 and v1beta3 since the expected strings were very different for those versions.